api/server/router/image/image_routes.go

```diff
@@ -95,7 +95,7 @@ func (ir *imageRouter) postImagesCreate(ctx context.Context, w http.ResponseWrit
 		}
 
 		if err := validateRepoName(ref); err != nil {
-			return errdefs.Forbidden(err)
```

Comment:
If we want to change `Forbidden` into `InvalidParameter` we should also do it for this error:
https://github.com/moby/moby/blob/62b33a2604c0bacdd26bfbe7303cb6e2ed26d432/distribution/errors.go#L208-L214

Reply:
changed.